sandbox/src/main/java/org/dcsa/conformance/sandbox/ConformanceOrchestrator.java

```diff
@@ -439,6 +439,8 @@ public ObjectNode getScenarioStatus(String scenarioId) {
       }
       scenarioNode.put("confirmationRequired", nextAction.isConfirmationRequired());
       scenarioNode.put("inputRequired", nextAction.isInputRequired());
+    } else {
+      scenarioNode.put("inputRequired", false);
```

Comment:
Conclusion: I revert this change, because it is scary to make changes in this class, without having proper testing in place and I can work around it in test code. Later, it is ok to make it better (and more consistent e.g.).